src/protocols/aeternity/libs/AeternityAdapter.ts

```diff
@@ -397,7 +397,10 @@ export class AeternityAdapter extends BaseProtocolAdapter {
           .filter(({ tx }) => !tx.contractId || tx.contractId === this.coinContractId);
       } else {
         regularTransactions = res.regularTransactions
-          .filter(({ tx }) => tx.contractId === assetContractId);
+          .filter(({ tx }) => (
+            tx.contractId === assetContractId
+            || JSON.stringify({ ...tx.arguments, ...tx.log }).search(assetContractId) !== -1
```

Comment:
I suggest to use `includes` instead of `search` as search is slower and the result needs to be compared with `-1`.